ftest-toxic-reporter/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-cube-q</artifactId>
+    <groupId>org.arquillian.cube.q</groupId>
+    <version>1.0.0.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-cube-q-reporter-ftest-toxic</artifactId>
```

Comment:
I have created separate project to test reporter functionality in isolation exactly same as how user is going to configure. We can configure it in the same module. The idea here to keep in separate module to test just reporter functionality. I don't want to pollute existing test.